src/services/analytics.js

```diff
@@ -0,0 +1,27 @@
+import generateFakeData from './dataGenerator';
+
+const getAnalytics = (dates) => {
```

Comment:
Yes this was my suggested name but i think getAnalyticsData will be better

Reply:
renamed function to `getAnalyticsData`